.travis.yml

```diff
@@ -19,8 +19,7 @@ services:
 addons:
   chrome: stable
 before_install:
-  - gem update --system
-  - gem install bundler --no-document
+  - gem update bundler
```

Comment:
Should we add the `--no-document` flag??

Reply:
Great shout, thanks for doing the digging 👍